pkg/cmd/config/config.go

```diff
@@ -3,7 +3,6 @@ package config
 import (
 	initcmd "github.com/coreeng/corectl/pkg/cmd/config/init"
 	"github.com/coreeng/corectl/pkg/cmd/config/set"
-	"github.com/coreeng/corectl/pkg/cmd/config/update"
```

Comment:
So we should be getting rid of the update command altogether, and every command that interacts with this repos, should perform a repo update.
No need for the update command any more then, users don't need to manually refresh repositories.